docs/high-availability/kube-controller-manager.manifest

```diff
@@ -0,0 +1,100 @@
+{
+"apiVersion": "v1beta3",
+"kind": "Pod",
+"metadata": {"name":"kube-controller-manager"},
+"spec":{
+"hostNetwork": true,
+"containers":[
+    {
+    "name": "kube-controller-manager",
+    "image": "gcr.io/google_containers/kube-controller-manager:fda24638d51a48baa13c35337fcd4793",
+    "command": [
+                 "/bin/sh",
+                 "-c",
+                 "/usr/local/bin/kube-controller-manager --master=192.168.4.102:8080 --service_account_private_key_file=/srv/kubernetes/server.key --v=4  1>>/var/log/kube-controller-manager.log 2>&1"
```

Comment:
`--master=192.168.4.102:8080` will only work with controller manager on one host since apiserver doesn't bind to all interfaces. (scheduler uses 127.0.0.1)